### ISSUE TITLE
About the F1-score of NYT24

### ISSUE THREAD

**Issue by xyzhao01** [nusnlp/PtrNetDecoding4JERE#7]
Hi,
I use the NYT24 dataset and code you published, and the best result obtained is 0.693, which is very different from the result of 0.817 in your paper. 
I want to know how to get the performance in your paper. Is the hyperparameter inappropriate? Or other reasons?

thank you.

**Comment by nayakt** [comment]
There is not any hyperparameter to tune. The release code should reproduce the results if you use the the released embeddings and files as it is. F1 score of 0.693 looks similar to the F1 score on the NYT29 dataset. Are you sure you are using the NYT24 not the NYT29?

**Comment by xyzhao01** [comment]
The dataset should be correct. Before that, I have checked the dataset(NYT24 with 24 relations) and w2v.txt, and I trained twice. Can you provide me with the model files trained on the NYT24 dataset? I want to find the problem :)  , xy.zhao@siat.ac.cn

**Comment by nayakt** [comment]
Which model you are trying to run ? Word level decoding one or pointer network decoding one ?

**Comment by xyzhao01** [comment]
Word level decoding one. Thank you!
I am trying on the word level decoding one，and use the same release file(word_decoder.py)

**Comment by nayakt** [comment]
I uploaded the pre-trained word decoding model for NYT24 dataset in the google drive link where data files are kept for the same dataset. You can find the link in the README.

**Comment by xyzhao01** [comment]
I really appreciate it！I will try it soon :)

**Comment by xyzhao01** [comment]
Thank you for your help! I understand the problem :)
① I used the pre-trained model you published, and the result measured on the NYT24 dataset is 0.81, which is consistent with the result in the paper.
This helped me confirm that the NYT24 dataset I used is correct.
② I run multiple times locally, and the result is about 0.69.
③ After confirming your results and my local results, I found that the dev dataset F1 score using "train" mode is 0.69, and the test dataset F1 score using "test" mode is 0.80.

Where is the problem？
It is glad to find why the F1 score between dev and test dataset is quite different, because the evaluation indicator F1 is calculated differently.  so I'm curious why don't use the same function to calculate F1? (I mean that the F1 indicator calculation method of test datasets can be used in the calculation method of train and dev datasets)

**Comment by nayakt** [comment]
On test dataset F1 is calculated based on the tuples. You can read evaluation metric part of my paper for this. This is standard evaluation for this task. The dev dataset is used just to choose the best model during training. So I used a word level F1 score. So it won't matter much. But yes, you can use the same F1 score for dev data too.

**Comment by xyzhao01** [comment]
Thank you so much！
